verbs.js

```diff
@@ -4,150 +4,175 @@
       },
       "attended" : {
          "id" : "http://adlnet.gov/expapi/verbs/attended",
          "display" : {"de-DE" : "nahm teil an",
                       "en-US" : "attended",
+                      "fr-FR" : "vue",
                       "es-ES" : "asistió"}
       },
       "commented" : {
          "id" : "http://adlnet.gov/expapi/verbs/commented",
          "display" : {"de-DE" : "kommentierte",
                       "en-US" : "commented",
+                      "fr-FR" : "commenté",
                       "es-ES" : "comentó"}
       },
       "completed" : {
          "id" : "http://adlnet.gov/expapi/verbs/completed",
          "display" : {"de-DE" : "beendete",
                       "en-US" : "completed",
+                      "fr-FR" : "terminé",
                       "es-ES" : "completó"}
       },
       "exited" : {
          "id" : "http://adlnet.gov/expapi/verbs/exited",
          "display" : {"de-DE" : "verließ",
                       "en-US" : "exited",
+                      "fr-FR" : "quitté",
                       "es-ES" : "salió"}
       },
       "experienced" : {
          "id" : "http://adlnet.gov/expapi/verbs/experienced",
          "display" : {"de-DE" : "erlebte",
                       "en-US" : "experienced",
+                      "fr-FR" : "testé",
                       "es-ES" : "experimentó"}
       },
       "failed" : {
          "id" : "http://adlnet.gov/expapi/verbs/failed",
          "display" : {"de-DE" : "verfehlte",
                       "en-US" : "failed",
+                      "fr-FR" : "échoué",
                       "es-ES" : "fracasó"}
       },
       "imported" : {
          "id" : "http://adlnet.gov/expapi/verbs/imported",
          "display" : {"de-DE" : "importierte",
                       "en-US" : "imported",
+                      "fr-FR" : "importé",
                       "es-ES" : "importó"}
       },
       "initialized" : {
          "id" : "http://adlnet.gov/expapi/verbs/initialized",
          "display" : {"de-DE" : "initialisierte",
                       "en-US" : "initialized",
+                      "fr-FR" : "initialisé",
                       "es-ES" : "inicializó"}
       },
       "interacted" : {
          "id" : "http://adlnet.gov/expapi/verbs/interacted",
          "display" : {"de-DE" : "interagierte",
                       "en-US" : "interacted",
+                      "fr-FR" : "interagis",
                       "es-ES" : "interactuó"}
       },
       "launched" : {
          "id" : "http://adlnet.gov/expapi/verbs/launched",
          "display" : {"de-DE" : "startete",
                       "en-US" : "launched",
+                      "fr-FR" : "lancé",
                       "es-ES" : "lanzó"}
       },
       "mastered" : {
          "id" : "http://adlnet.gov/expapi/verbs/mastered",
          "display" : {"de-DE" : "meisterte",
                       "en-US" : "mastered",
+                      "fr-FR" : "answered",
```

Comment:
Forgive me, I don't know French. I just wanted to confirm this is the right translation. It just caught my eye as a possible copy/paste error.